### PR TITLE
2911 update health check sitemap links

### LINKED ISSUE #2911: 🔗 Footer - fix Health Check/Sitemap links
### Description

The health check/sitemap links in the footer point to the health check/sitemap form the asp.net server. These pages are no longer accessible because Cloudflare worker is sitting in from of the route to the old server.

```[tasklist]
- [x] Add a todo page page for adding a HealthCheck for the website
- [x] Add a todo page for adding a Sitemap (not to be mistaken with the sitemap used by Google to index our pages)
- [x] Update the HealthCheck in our footer to go to the todo page (/healtcheck)
- [x] Update the Sitemap link in our footer to go to the todo page (/sitemap)
```

### Screenshot
Image

**Figure**: **Footer for SSW Website**

Image

### PR DESCRIPTION
**Pull request by Calinator444**
### Description

The footer links for the Health Check and Sitemap pages are currently 404ing. I've added a couple of placeholder pages and am planning to add redirects to the new pages.

- 🧪 fixed playwright test failing
- 🚧 Added placeholder page for the sitemap
- 🚧 Added placeholder page for the Menumap

- Affected routes: 
- ssw.com.au/*
- /sitemap
- /healtcheck

- Fixed #2911 

- [ ] Include done video or screenshots

image

**Figure**: **Placeholder page for Healthcheck**

image

**Figure**: **Placeholder page for Sitemap**

### CODE REVIEW

File: components/layout/footer/site-info.tsx [hunk 1]
```diff
@@ -6,14 +6,14 @@ export const SiteInfo = () => (
...
-      href="https://www.ssw.com.au/ssw/MenuMap.aspx"
+      href="/ssw/MenuMap.aspx"
     >
       SITEMAP
     </CustomLink>
     <Divider />
     <CustomLink
       // TODO: Implementation
-      href="https://www.ssw.com.au/ssw/HealthCheck"
+      href="/ssw/HealthCheck"
```
Comment (bradystroud) [inline components/layout/footer/site-info.tsx]: I know you are adding a redirect for these, but I think you should fix the URL at the source as well.

Comment (bradystroud) [reply components/layout/footer/site-info.tsx]: Also, there are TODOs in this code, please link them to a PBI

**Review by bradystroud — CHANGES_REQUESTED** [review]

File: components/layout/footer/site-info.tsx (hunk 1 above)
Comment (Calinator444) [reply components/layout/footer/site-info.tsx]: > I know you are adding a redirect for these, but I think you should fix the URL at the source as well.

Done - I've fixed the links at the source

> Also, there are TODOs in this code, please link them to a PBI

Done - I've added links to the pbis for implementing these page

**Comment by Calinator444** [comment]
I had @tiagov8 Approved the designs on these pages

File: content/pages/sitemap.mdx [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+
+      TODO: [✨ Sitemap - Add a page compiling all links on the
+      site](https://github.com/SSWConsulting/SSW.Website/issues/2913 "✨ Sitemap
+      - Add a page compiling all links on the site")
+
+
+      For hot tips about our best practices go to [SSW Rules]()
+
+
+      Learn more about our employees on [SSW People](/people "SSW People")
```
Comment (bradystroud) [inline content/pages/sitemap.mdx]:
```suggestion
      Check out all our best practices on [SSW Rules](/rules)


      Learn more about our employees on [SSW People](/people "SSW People")
```

**Review by bradystroud — CHANGES_REQUESTED** [review]

File: content/pages/sitemap.mdx (hunk 2 above)
Comment (Calinator444) [reply content/pages/sitemap.mdx]: Test passed by @sethdaily

**Review by bradystroud — APPROVED** [review]
LGTM

**Pull request merged**